core/src/main/java/nl/nn/adapterframework/processors/ExceptionHandlingPipeProcessor.java

```diff
@@ -47,6 +47,7 @@ protected PipeRunResult processPipe(PipeLine pipeLine, IPipe pipe, Message messa
 
 				ErrorMessageFormatter emf = new ErrorMessageFormatter();
 				Message errorMessage = emf.format(e.getMessage(), e.getCause(), pipeLine.getOwner(), message, pipeLineSession.getMessageId(), tsReceivedLong);
+				log.warn("exception occured, forwarding to exception-forward [{}], exception:\n", PipeForward.EXCEPTION_FORWARD_NAME, e);
```

Comment:
I'm not sure loglevel WARN is desirable here, when there is an exception-forward we are already dealing with the exception inside the adapter. Depending on how the adapter is designed, this could add a lot of noise in logs for situations that do not need any attention from operations.

I would suggest level INFO instead.